go/example_code/rekognition/detect_faces.go

```diff
@@ -0,0 +1,106 @@
+// snippet-comment:[These are tags for the AWS doc team's sample catalog. Do not remove.]
+// snippet-sourceauthor:[Doug-AWS]
+// snippet-sourcedescription:[Deletes the website configuration on an S3 bucket.]
+// snippet-keyword:[Amazon Simple Storage Service]
+// snippet-keyword:[Amazon S3]
+// snippet-keyword:[DeleteBucketWebsite function]
+// snippet-keyword:[Go]
+// snippet-sourcesyntax:[go]
+// snippet-service:[s3]
+// snippet-keyword:[Code Sample]
+// snippet-sourcetype:[full-example]
+// snippet-sourcedate:[2018-03-16]
+/*
+   Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   This file is licensed under the Apache License, Version 2.0 (the "License").
+   You may not use this file except in compliance with the License. A copy of
+   the License is located at
+    http://aws.amazon.com/apache2.0/
+   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+   specific language governing permissions and limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/rekognition"
+)
+
+var svc *rekognition.Rekognition
+var bucket = flag.String("bucket", "", "The name of the bucket")
+var photo = flag.String("photo", "", "The path to the photo file (JPEG, GIF, or ???)")
```

Comment:
Rekognition only supports JPG and PNG files.

Reply:
Updated message.